packages/turf-boolean-parallel/test.js

```diff
@@ -13,7 +13,6 @@ test('turf-boolean-parallel', t => {
         const [line1, line2] = geojson.features;
         const result = booleanParallel(line1, line2);
 
-        if (process.env.SHAPELY) shapely.contains(line1, line2).then(result => t.true(result, '[true] shapely - ' + name));
```

Comment:
@DenisCarriere why did you remove this?
Many other boolean modules have that, some of them also have
```js
if (process.env.JSTS) t.false(booleanJSTS('within', feature1, feature2), '[false] JSTS - ' + name);
```
What's their use exactly?